conf/application.conf

```diff
@@ -58,10 +58,11 @@ play.http.router = prod.Routes
 controllers {
 }
 
-# Microservice specific config
-
 mongodb {
   uri = "mongodb://localhost:27017/customs-service-status"
+  document {
+    expireAfterSeconds = 86400 //24h
```

Comment:
I think it should be hash character for comment (# 24h) instead of double forward slash (//24h)

Reply:
actually scratch that, just checked, // works

Reply:
// also works , it should be # in the app-config yaml files